src/components/Foldermenu/index.tsx

```diff
@@ -162,7 +181,7 @@ function Foldermenu() {
           targetName={activeButton}
           onClose={handleCloseShareModal}
           isModalOpen={isModalShareOpen}
-          currentFolderId={currentFolderId} // currentFolderId 전달
+          currentFolderId={currentFolderId || undefined}
```

Comment:
```suggestion
          currentFolderId={currentFolderId}
```
|| undefined 를 주시는 것은 안주시는 것과 동일해요! (Id 가 0 일 때 undefined를 할당하고 싶으신 게 아니라면)